set_RMQ_EC2.py

```diff
@@ -0,0 +1,69 @@
+##Should run on local machine
```

Comment:
Refer to https://www.python.org/dev/peps/pep-0008
Comments should start with one `#`